src/main/java/at/ac/tuwien/kr/alpha/config/CommandLineParser.java

```diff
@@ -335,16 +335,18 @@ private void handleBranchingHeuristic(Option opt, SystemConfig cfg) throws Parse
 		try {
 			cfg.setBranchingHeuristicName(branchingHeuristicName);
 		} catch (IllegalArgumentException e) {
-			throw new ParseException("Unknown branching heuristic: " + branchingHeuristicName + ". Please try one of the following: " + Heuristic.listAllowedValues());
```

Comment:
Please revert to the old formatting here.

Reply:
done